classes.py

```diff
@@ -28,9 +34,29 @@ class ChangeSet():
     """Pages that were added in the diff"""
     removed: List[Page]
     """Pages that were removed in the diff"""
-    changed: List[Tuple[Page, List[PropertyChange]]]
+    changed: Dict[Page, Tuple[List[PropertyChange], datetime]]
     """Pages that had their properties changed in the diff. The key is the page id and the value is the list of 
     properties that were changed"""
 
     def is_empty(self) -> bool:
         return len(self.added) == 0 and len(self.removed) == 0 and len(self.changed) == 0
+
+    def merge_set(self, other_changed_set):
```

Comment:
Add type annotations

Reply:
How? It wont let me do typehint to ChangeSet